benchmarks/timing.h

```diff
@@ -0,0 +1,13 @@
+// Timers for use in benchmarking.
+
+#pragma once
+
+#include <time.h>
+
+uint64_t Now() {
+  timespec t;
+  if (clock_gettime(CLOCK_MONOTONIC, &t)) {
```

Comment:
I'm fine with this, but it's worth noting that this won't work on MacOS.  In the longer term, this code should use c++11's high_resolution_clock.  (I'd love it if you'd switch it sooner, of course, because then I could compile on my laptop. :-).  But also fine to get things working and clear this bit up later when we do a portability pass.